libraries/core_libs/network/src/tarcap/packets_handlers/vote_packet_handler.cpp

```diff
@@ -63,7 +63,9 @@ void VotePacketHandler::process(const PacketData &packet_data, const std::shared
     peer->markPbftBlockAsKnown(pbft_block->getBlockHash());
   }
 
-  processVote(vote, pbft_block, peer, true);
+  if (processVote(vote, pbft_block, peer, true)) {
+    return;
```

Comment:
Why return if vote was processed successfully?

Reply:
yeah thats bug - I wonder how is it possible that it works on devnet